distribution/lib/Standard/Base/0.0.0-dev/src/Data/Ref.enso

```diff
@@ -0,0 +1,45 @@
+## Utilities for working with mutable references.
+type Ref
+
+    ## A mutable reference type.
+    @Builtin_Type
+    type Ref
+
+## FIXME consider moving this as methods of Ref, currently a lot of stuff depends
+##       on the current design
```

Comment:
A bit hesitant of merging a _FIXME_. Maybe make it a TODO? I guess it would also be best to create a Pivotal ticket for this task and link to it here. Otherwise will be very easy to forget about this.

Reply:
Yeah, it should have been a TODO

Reply:
Agreed https://www.pivotaltracker.com/story/show/182026558